src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -2837,6 +2837,7 @@ static void _svgLoaderParserXmlCssStyle(SvgLoaderData* loader, const char* conte
         length -= next - content;
         content = next;
 
+        free(node->id);
         free(tag);
```

Comment:
It looks wrong position to free it. I believe @mgrudzinska could help this.

Reply:
@patrykka as far as I see the node (and everything what it contains, like id) is freed in the ```_freeNode```, but I may be missing something. could you please provide an example in which you observed mem leak?